android/app/src/main/AndroidManifest.xml

```diff
@@ -38,19 +38,47 @@
         <action android:name="android.intent.action.MAIN" />
         <category android:name="android.intent.category.LAUNCHER" />
       </intent-filter>
+      <!-- "Share" action -->
       <intent-filter>
         <action android:name="android.intent.action.SEND" />
+
         <category android:name="android.intent.category.DEFAULT" />
-        <data android:mimeType="*/*" />
+
+        <data android:mimeType="application/pdf" />
+        <data android:mimeType="text/*" />
+        <data android:mimeType="image/*" />
+        <data android:mimeType="application/x-scilab" /> <!-- for .sce -->
+        <data android:mimeType="application/octet-stream" /> <!-- fallback for other ASiC files -->
+        <data android:mimeType="application/xhtml+xml" />
```

Comment:
existuje aj `application/xml`. Potom ešte máme na Slovensku takúto špecialitu: https://www.iana.org/assignments/media-types/application/vnd.gov.sk.xmldatacontainer+xml  dal by sa nejako pridať ten mimetype? Alebo to rieši systém a ak to nebude poznať, tak tam dá XML? (XML je tiež ok pre to)

Reply:
Ten application/xml pridam, aj ked funguje to uz teraz na otvaranie XML.

Mime types resolvuje Android na zaklade pripony (preto je tam aj specialny case pre ".sce").
Existuje use case, ze by som rovno z **webu** stahoval **XML Data Container** s takym `Content-type`?

Reply:
v praxi nie. Nemusíme to tým pádom hrotiť.